pomegranate/gmm.pyx

```diff
@@ -305,9 +285,12 @@ cdef class GeneralMixtureModel( Model ):
 		cdef double log_probability_sum = NEGINF
 		cdef double log_probability
 
-		for i in range( self.n ):
-			log_probability = ( <Model> self.distributions_ptr[i] )._log_probability(X) + self.weights_ptr[i]
-			log_probability_sum = pair_lse( log_probability_sum, log_probability )
+		for i in xrange(self.n):
```

Comment:
There's no need to use xrange in cython code because it'll optimize range from a python loop to a C style loop.